packages/flutter_reactive_ble/CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 5.2.0
+
+* Bumps to Dart 2.17 minimum and upgrades melos to 3.1.0 in #762
```

Comment:
The other lines are in imperative form (e.g. "Update...", "Bump..."). Consider making this line consistent with the others.